modules/ocf_desktop/files/stats/staff-visibility.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+import sys
+import os
+import re
+
+LAUNCHER_SETTINGS_PATH = os.path.expanduser('~/.config/xfce4/panel/launcher-21/switch.desktop')
+UPDATE_FLAGS_SCRIPT_PATH = os.path.expanduser('/opt/stats/update-flags.sh')
+
+def changePanelState(file_path, icon_name, visibility):    
+    with open(file_path, 'r') as f:
+        content = f.read()
+    content_1 = re.sub(r'((?<=Icon=).*)', icon_name, content)
```

Comment:
Looks good. It's a shame that `configparser` won't work on regular `ini` files because they don't have sections.